data/saas/config/shipstation_config.yml

```
@@ -0,0 +1,74 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Shipstation
+  type: shipstation
+  description: A sample schema representing the Shipstation connector for Fides
+  version: 0.1.1
+
+  connector_params:
+    - name: api_key
+    - name: api_secret
+
+  external_references:
+    - name: customer_id
+      label: customerId Field
+      description: The dataset reference to the Shipstation customerId
```

Any UI-facing value should be in "sentence case", where we only capitalize the first letter, acronyms, or proper nouns

```suggestion
      label: Customer ID field
      description: The dataset reference to the Shipstation customer ID
```